cfgov/jinja2/v1/_includes/organisms/email-popup/oah.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% set popup_headline = "Buying a home?" %}
+{% set popup_content = "Sign up for email tips and info to help you through the process." %}
+{% set popup_image = "img/modal-house.png" %}
+{% set mailing_list_code = "USCFPB_128" %}
```

Comment:
Trivial: it would be nice if the variable order was the same in debt.html and oah.html

Reply:
@anselmbradford thanks, addressed in 327e494.